src/typechecker/FStar.TypeChecker.TcInductive.fs

```diff
@@ -214,7 +214,7 @@ let tc_data (env:env_t) (tcs : list<(sigelt * universe)>)
 (*close*)let t = U.arrow ((tps |> List.map (fun b -> {b with binder_qual=Some (Implicit true)}))@arguments) (S.mk_Total result) in
                         //NB: the tps are tagged as Implicit inaccessbile arguments of the data constructor
          let t = SS.close_univ_vars _uvs t in
-         { se with sigel = Sig_datacon(c, _uvs, t, tc_lid, ntps, []) },
+         { se with sigel = Sig_datacon(c, _uvs, t, tc_lid, ntps, mutual_tcs) },
```

Comment:
The other implementation change is this:

We were dropping the list of mutually defined inductive types in the last field of a Sig_datacon

Reply:
Wow, must have been a pain to find after the first iteration of the change would have failed because of no mutuals?